src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -131,6 +134,8 @@ public Object instanceMainPostParseArgs() {
         this.defaultHeaders.add(new StringHeader("Started on: " +
                                 startDateTime.format(DateTimeFormatter.ofLocalizedDateTime(FormatStyle.LONG))));
 
+        LoggingUtils.sendLogToFile(logToFile);
+
         LoggingUtils.setLoggingLevel(VERBOSITY);  // propagate the VERBOSITY level to logging frameworks
 
         for (final File f : TMP_DIR) {
```

Comment:
There are a few messages that are output that are sent directly to stdout here which should also be included in a log file.  

ex:
```
[July 10, 2017 10:49:51 AM EDT] org.broadinstitute.hellbender.tools.PrintReads  --output output.bam --input src/test/resources/NA12878.chr17_69k_70k.dictFix.bam  --interval_set_rule UNION --interval_padd>
[July 10, 2017 10:49:51 AM EDT] Executing as louisb@WMD2A-31E on Mac OS X 10.11.6 x86_64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_112-b16; Version: Version:removeNativeBiuld-380-g4267ce3-SNAPSHOT
[July 10, 2017 10:49:52 AM EDT] org.broadinstitute.hellbender.tools.PrintReads done. Elapsed time: 0.01 minutes.
Runtime.totalMemory()=192937984
```